doc/users-guide/users-guide.adoc

```diff
@@ -1198,4 +1198,6 @@ include::users-guide-tm.adoc[]
 
 include::users-guide-cls.adoc[]
 
+include::users-guide-utilities-examples.adoc[]
```

Comment:
Having this as a section of the User Guide is fine, but you need to add this file to the list in `doc/users-guide/Makefile.am` so that changes to it will cause the User Guide to be regenerated.